client/app/scripts/liveblog-settings/module.js

```diff
@@ -0,0 +1,77 @@
+(function() {
+    'use strict';
+    LiveblogSettingsController.$inject = ['$scope', 'api', '$location', 'notify', 'gettext', '$q'];
+    function LiveblogSettingsController($scope, api, $location, notify, gettext, $q) {
+        //prep the settings
+        $scope.liveblogSettings = {'language': {}, 'theme': {}};
+        $scope.settingsLoading = true;
+        api.languages.query().then(function(data) {
+            $scope.languages = [];
+            _.map(data._items, function(language) {
+                $scope.languages.push(language);
+            });
+        });
+        api.themes.query().then(function(data) {
+            $scope.themes = [];
+            _.map(data._items, function(theme) {
+                $scope.themes.push(theme);
+            });
+        });
+        $scope.settingsLoading = true;
```

Comment:
This line is already defined line 7.
Actually, if you were using `angular.extend($scope, {})` to define your scope, jsLint would told you that you have a duplicated definition of `settingsLoading`.